modules/net-lb-int/variables.tf

```diff
@@ -36,6 +36,7 @@ variable "backend_service_config" {
       ratio                     = optional(number)
     }))
     log_sample_rate  = optional(number)
+    protocol         = optional(string, "UNSPECIFIED")
```

Comment:
nit: if UNSPECIFIED is the default in the provider, I would use null as the default in the optional